src/SpectatorView.Unity/Assets/SpectatorView/Scripts/UI/MobileNetworkConfigurationVisual.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Net;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Microsoft.MixedReality.SpectatorView
+{
+    public class MobileNetworkConfigurationVisual : MonoBehaviour,
+        INetworkConfigurationVisual
+    {
+        [Tooltip("The InputField used to specify an IPAddress by the user.")]
+        [SerializeField]
+        private InputField ipAddressField = null;
+
+        [Tooltip("The Button used to start a network connection by the user.")]
+        [SerializeField]
+        private Button connectButton = null;
+
+        [Tooltip("The text used to display error messages to the user.")]
+        [SerializeField]
+        private Text errorText = null;
+
+        [Tooltip("Check to enable debug logging.")]
+        [SerializeField]
+        private bool debugLogging = false;
+
+        private string ipAddress = "127.0.0.1";
+
+        public event NetworkConfigurationUpdatedHandler NetworkConfigurationUpdated;
+
+        private void OnEnable()
+        {
+            if (connectButton != null)
+            {
+                connectButton.onClick.AddListener(OnConnectButtonClick);
+            }
+
+            ipAddressField.text = ipAddress;
```

Comment:
>ipAddressField.text = ipAddress; [](start = 12, length = 32)

It would be nice to use PlayerPrefs to store and remember the previously-entered IP address so that you don't have to type it every time